src/plugins/foxPage/foxPage.tsx

```diff
@@ -41,29 +47,28 @@ export const FoxPage = (props: FoxPageProps) => {
   const history = useHistory()
   const assetFox = useAppSelector(state => selectAssetById(state, FoxAssetId))
   const assetFoxy = useAppSelector(state => selectAssetById(state, FoxyAssetId))
-  const isFoxSelected = props.activeAssetId === FoxAssetId
-  const isFoxySelected = props.activeAssetId === FoxyAssetId
-  const selectedAsset = isFoxSelected ? assetFox : assetFoxy
+
+  const assets = useMemo(() => [assetFox, assetFoxy], [assetFox, assetFoxy])
+
+  const selectedAssetIndex = useMemo(
+    () => assets.findIndex(asset => asset.assetId === props.activeAssetId),
+    [props.activeAssetId, assets],
+  )
+
+  const selectedAsset = assets[selectedAssetIndex]
+
   const [isLargerThanMd] = useMediaQuery(`(min-width: ${breakpoints['md']})`)
   const mobileTabBg = useColorModeValue('gray.100', 'gray.750')
   const { description } = assetFox || {}
   const query = useGetAssetDescriptionQuery(FoxAssetId)
   const isLoaded = !query.isLoading
 
-  const handleFoxClick = () => {
-    if (isFoxSelected) {
+  const handleTabClick = (assetId: AssetId) => {
+    if (assetId === props.activeAssetId) {
```

Comment:
stylistically, i *hate* seeing `props.` in components this days, would be nice if we could destructure, but this is a nitpick